src/js/components/Articles/modules/EpubView/style.js

```diff
@@ -0,0 +1,10 @@
+export default {
```

Comment:
is this supposed to stylesheet?

Reply:
no its the layout of the ereader container